core/src/com/unciv/models/ruleset/unique/UniqueType.kt

```diff
@@ -835,6 +835,7 @@ enum class UniqueType(
     OneTimeGainStatRange("Gain [amount]-[amount] [stat]", UniqueTarget.Triggerable, flags = setOf(UniqueFlag.AcceptsSpeedModifier)),
     OneTimeGainPantheon("Gain enough Faith for a Pantheon", UniqueTarget.Triggerable),
     OneTimeGainProphet("Gain enough Faith for [amount]% of a Great Prophet", UniqueTarget.Triggerable),
+    Eureka("Research [relativeAmount]% of [tech]", UniqueTarget.Triggerable),
```

Comment:
Rename to OneTimeGainTechPercent
OneTime - prefix for all triggerables
GainTechPercent - describes what it does